packages/salesforcedx-vscode-core/src/commands/forceApexTestRun.ts

```diff
@@ -107,9 +114,10 @@ export class ForceApexTestRunCommandFactory {
         );
         break;
       case TestType.Class:
-        this.builder = this.builder
-          .withFlag('--classnames', `${this.data.label}`)
-          .withArg('--synchronous');
```

Comment:
If I remember correctly we're removing this because it doesn't allow us to get the results in a json file. If that's true, did we ask the cli team about that behavior ?

Reply:
Yeah, there is a bug with that functionality; including the synchronous argument should still result in a json file creation. I confirmed it was a bug with the cli team and filed it in gus.